ImageProcessing/Re-RunHSC.ipynb

```diff
@@ -0,0 +1,271 @@
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!eups list lsst_distrib\n",
+    "datarepo = \"/home/jmyles/repositories/ci_hsc/\"\n",
+    "datadir = \"/home/jmyles/DATA/\"\n",
+    "os.system(\"mkdir -p {}\".format(datadir))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!setup -j -r /home/jmyles/repositories/ci_hsc\n",
+    "\n",
+    "setup = eups.setupcmd.EupsSetup([\"-j\",\"-r\", datarepo])\n",
+    "status = setup.run()\n",
+    "print('setup exited with status {}'.format(status))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A Butler needs a *mapper* file \"to find and organize data in a format specific to each camera.\" We write this file to the data repository so that any instantiated Butler object knows which mapper to use."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open(datadir + \"_mapper\", \"w\") as f:\n",
+    "    f.write(\"lsst.obs.hsc.HscMapper\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ingest script\n",
+    "!ingestImages.py /home/jmyles/DATA /home/jmyles/repositories/ci_hsc/raw/*.fits --mode=link"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!installTransmissionCurves.py /home/jmyles/DATA\n",
+    "\n",
+    "from lsst.obs.hsc import makeTransmissionCurves, HscMapper\n",
+    "from lsst.daf.persistence import Butler\n",
+    "\n",
+    "butler = Butler(outputs={'root': datadir, 'mode': 'rw', 'mapper': HscMapper})\n",
+    "\n",
+    "for start, nested in makeTransmissionCurves.getFilterTransmission().items():\n",
+    "    for name, curve in nested.items():\n",
+    "        if curve is not None:\n",
+    "            butler.put(curve, \"transmission_filter\", filter=name)\n",
+    "for start, nested in makeTransmissionCurves.getSensorTransmission().items():\n",
+    "    for ccd, curve in nested.items():\n",
+    "        if curve is not None:\n",
+    "            butler.put(curve, \"transmission_sensor\", ccd=ccd)\n",
+    "for start, curve in makeTransmissionCurves.getOpticsTransmission().items():\n",
+    "    if curve is not None:\n",
+    "        butler.put(curve, \"transmission_optics\")\n",
+    "for start, curve in makeTransmissionCurves.getAtmosphereTransmission().items():\n",
+    "    if curve is not None:\n",
+    "        butler.put(curve, \"transmission_atmosphere\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ingest calibration images into Butler repo\n",
```

Comment:
When explaining the calib files, I wonder if it's worth including a few `ls` commands to show the user what is available in teh data repo. Same for the science images. Or, you could refer to the Basics/DataInventory tutorial and have the user go look at that instead.